ner/ner_eval_ab.py

```diff
@@ -0,0 +1,160 @@
+from collections import Counter
+from typing import List, Optional
+import random
+from tabulate import tabulate
```

Comment:
This is not a built-in, so people will have to install it, right? That's a bit inconvenient so maybe we can use `wasabi` instead, which has utilities for rendering a table and will come pre-installed by default with Prodigy, since it's a depencency.

Reply:
Done.